### DISCUSSION TITLE
Support the dialog element with the modal component

### DISCUSSION THREAD

**Discussion by hahn-kev** [saadeghi/daisyui#1714] [Ideas / feature requests]
The [dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) has support from all the major browsers these days at 93% on [caniuse](https://caniuse.com/dialog). 

However in my testing daisyui doesn't play well with it and there's no documentation for how to use the 2 together which would be helpful.

Could daisyui support using the dialog element in both `show` and `showModal` modes? In my case I had to work around an issue with `modal-bottom` not working, but I figured a discussion about supporting the element would be appreciated as it would solve accessibility issues like focus trapping as mentioned in #1662 and #1609.

**Comment by saadeghi** [comment]
Yes, I'm planning to use dialog element for daisyUI v3.
